airflow/config_templates/default_airflow_local_settings.py

```diff
@@ -26,17 +26,6 @@
 
 BASE_LOG_FOLDER = conf.get('core', 'BASE_LOG_FOLDER')
 
-# TODO: REMOTE_BASE_LOG_FOLDER should be deprecated and
-# directly specify in the handler definitions. This is to
-# provide compatibility to older remote log folder settings.
-REMOTE_BASE_LOG_FOLDER = conf.get('core', 'REMOTE_BASE_LOG_FOLDER')
-S3_LOG_FOLDER = ''
```

Comment:
I think for 1.9 we should have this (and others in this diff) still be present but deprecated.